app/webpacker/stylesheets/components/_card.scss

```diff
@@ -257,14 +257,18 @@
   justify-content: space-between;
 }
 
-.card--dashboard--cs-accelerator,
+.card--dashboard--accelerator,
```

Comment:
was this change deliberate? I think this class is used in /app/views/dashboard/certificates/_cs-accelerator.html.erb